src/Tribe/Importer/File_Importer_Organizers.php

```diff
@@ -36,22 +36,105 @@ protected function create_post( array $record ) {
 		return $id;
 	}
 
+	/**
+	 * Build a organizer array for creation/update of the current imported organizer.
+	 *
+	 * @since 3.2
+	 * @since TBD Adjust to prevent overwriting values that aren't mapped.
+	 *
+	 * @param int    $organizer_id The ID of the organizer we're currently importing
+	 * @param array  $record       An event record from the import.
+	 *
+	 * @return array $organizer The array of organizer data for creation/update.
+	 */
 	private function build_organizer_array( $organizer_id, array $record ) {
-		$organizer              = array(
-			'Organizer'     => $this->get_value_by_key( $record, 'organizer_name' ),
-			'Description'   => $this->get_value_by_key( $record, 'organizer_description' ),
-			'Email'         => $this->get_value_by_key( $record, 'organizer_email' ),
-			'Phone'         => $this->get_value_by_key( $record, 'organizer_phone' ),
-			'Website'       => $this->get_value_by_key( $record, 'organizer_website' ),
-			'FeaturedImage' => $this->get_featured_image( $organizer_id, $record ),
+		$organizer          = [];
+		$columns        = [
+			'Organizer'   => 'organizer_name',
+			'Description' => 'organizer_description',
+			'Email'       => 'organizer_email',
+			'Phone'       => 'organizer_phone',
+			'Website'     => 'organizer_website',
+		];
+
+		foreach ( $columns as $name => $key ) {
+			// Reset.
+			$value = '';
+
+			// Don't set/overwrite unmapped columns.
+			if ( ! $this->has_value_by_key( $record, $key ) ) {
+				continue;
+			}
+
+			/**
+			 * Allows filtering of main values before setting.
+			 * Return boolean false to prevent importing that value.
+			 * @since TBD
+			 *
+			 * @param string $key       The key for the value we'll be importing.
+			 * @param string $value     The value we'll be importing.
+			 * @param array  $organizer The array of organizer data we're modifying.
+			 * @param array  $record    The event record from the import.
+			 */
+			$value = apply_filters(
+				"tribe_events_importer_organizer_{$key}_value",
+				$this->get_value_by_key( $record, $key ),
+				$key,
+				$organizer,
+				$record,
+				$this
+			);
+
+			if ( false === $value ) {
+				continue;
+			}
+
+			$organizer[ $name ] = $value;
+		}
+
+		// Handle the manual stuff.
+		$organizer['FeaturedImage'] = $this->get_featured_image( $organizer, $record );
+
+		/**
+		 * Allows filtering of record values before import.
+		 * Deprecated to match filter naming conventions.
+		 *
+		 * @since 4.2
+		 * @deprecated TBD
+		 *
+		 * @param array $organizer The array of organizer data we're modifying.
+		 * @param array $record The event record from the import.
+		 * @param int   $organizer_id The ID of the organizer we're currently importing.
+		 */
+		$organizer = apply_filters_deprecated(
+			'tribe_events_csv_import_organizer_fields',
+			[
+				$organizer,
+				$record,
+				$organizer_id,
+				$this
```

Comment:
* :no_entry_sign: Each array item in a multi-line array declaration must end in a comma
* :no_entry_sign: Comma required after last value in array declaration

<sub>hash: cf69707c01b4b0773254019e6cea32c8</sub>

Reply:
```suggestion
				$this,
```